app/models/user.rb

```diff
@@ -104,8 +104,8 @@ def student
 
   validates :name, :email, :country, :location, presence: true, unless: :github_import
 
-  accepts_nested_attributes_for :attendances, allow_destroy: true
-  accepts_nested_attributes_for :roles, allow_destroy: true
+  accepts_nested_attributes_for :attendances, allow_destroy: true, limit: 3
+  accepts_nested_attributes_for :roles
```

Comment:
For scoping this down to the current season, it should be as I already wrote in the conversation. However, if that's all too complex, what about "just" a validation? I imagine that something like
```ruby
validates_length_of :conferences_in_season, maximum: 3
```
should already be sufficient. Don't you think?